firebase-sessions/src/main/kotlin/com/google/firebase/sessions/SessionGenerator.kt

```diff
@@ -19,11 +19,11 @@ package com.google.firebase.sessions
 import java.util.UUID
 
 /**
- * [SessionInfo] is a data class responsible for storing information about the current Session.
+ * [SessionState] is a data class responsible for storing information about the current Session.
```

Comment:
Nitpick comment here. I prefer `SessionInfo` or `SessionDetails` over `SessionState`. Session state makes me feel this this talks about the state of the session where in state would mean - Created, embellished, cancelled, dropped, dispatched. But in reality, we are saving the details of the current session.

Just a preference here. If you feel too strong about `SessionState`, I can be convinced.

Reply:
Renamed to `SessionDetails`.